### PR TITLE
Set permissions on copied libcurl

### PR DESCRIPTION
**Pull request by markisaa**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,14 +25,15 @@ tests:
 	$(DC) -unittest $(LIB_FLAGS) $(FLAGS) $(TEST_SOURCES) -of$(TEST_PROGRAM)
 
 check: tests
-	chmod 555 $(LIBCURL)
 	cp $(LIBCURL) .
+	chmod 555 libcurl.dll
```
Comment (andralex) [inline Makefile]: This undoes the benefits of having the symbol `$(LIBCURL)`. Use this:

```
chmod 555 $(notdir $(LIBCURL))
```

File: Makefile [hunk 2]
```diff
@@ -25,14 +25,15 @@ tests:
...
 	./$(TEST_PROGRAM)
 
 install: driver check
 	mkdir -p $(TEMP)
 	cp $(PROGRAM) $(TEMP)/$(PROGRAM)
 	cp $(LIBCURL) $(TEMP)/
+	chmod 555 $(TEMP)/libcurl.dll
```
Comment (andralex) [inline Makefile]: ditto

**Comment by electrum** [comment]
Title the commit something like "Set permissions on copied libcurl"

**Comment by markisaa** [comment]
@electrum - Done. Merging.

**Pull request merged**